jetty-util/src/main/java/org/eclipse/jetty/util/SharedBlockingCallback.java

```diff
@@ -111,7 +112,7 @@ protected void notComplete(Blocker blocker)
      * callback do not blocak, rather they wakeup the thread that is blocked
      * in {@link #block()}
      */
-    public class Blocker implements Callback, Closeable
+    public class Blocker implements Callback, Closeable, Runnable
```

Comment:
Same as above: I do not like this mixing of concerns.

Reply:
Punted to a TODO

Reply:
See #7270